9.traverseTreeAlphabetical.js

```diff
@@ -25,7 +25,16 @@
 
 // Implement your code here
 const traverseTreeAlphabetical = (node, result = []) => {
+    if (!node) return null;
 
+    let current = [node]
+    if(current){
+        traverseTreeAlphabetical(node.left)
+        traverseTreeAlphabetical(node.right)
+        result.push(node.val)
+    }
+   
+    return result
 }
 
```

Comment:
Really close, you're doing a post order traversal when it needs to be in order. left node, current node, right node.